### ISSUE TITLE
Encoding error in dataset names and tags after migrating from version 3.0 to 3.4

### ISSUE THREAD

**Issue by RicardoOrtizG** [OpenRefine/OpenRefine#3511]
The names of dataset in Open Refine version 3.4 is showing encoding issues.

### To Reproduce
1. Migrate from 3.0 version to 3.4 version run in Windows 10 (64 bits) 
2. Display OpenRefine using Chrome browser.
3. Look for files previously created.

The files tags shows like this. 

![image](https://user-images.githubusercontent.com/15789737/105503931-2b397380-5c95-11eb-9690-ef0b1c134562.png)

And the file names

![image](https://user-images.githubusercontent.com/15789737/105504029-499f6f00-5c95-11eb-9c9c-e78d2592dfdb.png)

The original names of these datasets and tags have special characters from the Spanish Language like (Acompañamiento, Secretaría) but it doesn't fail when migrating from 2.8 version to 3.0 version, but from version 3.0 to 3.4 generates this error. 

Finally I come back to 3.0 version, but the files still showing the same encoding issue.

**Comment by tfmorris** [comment]
Before doing anything else, you should restore the backup that you made of your OpenRefine workspace before the attempted upgrade. OpenRefine 3.4 will have written out the project list using UTF-8 encoding, while 3.0 is trying to read it using your local encoding.

Version 3.4.1 (and 3.4, but you shouldn't be using it) include a bug fix for the `workspace.json` file not being correctly saved in UTF-8 format (#2657) which could be the cause of this.

If you could provide a copy of your `workspace.json` (either attach here or email to me at the same name on gmail), we can take a look and see what the issue is with the migration. Also please let us know how the locale is set on your Windows system.

We might need to provide a special upgrade procedure for Windows system which have a locale set that uses character encodings other than UTF-8.

**Comment by RicardoOrtizG** [comment]
Thanks for the answer, 
I tried to use the 3.4.1. version but I have the same encoding problem until it completely collapsed when I tried to delete the "bad" files.([#3514](https://github.com/OpenRefine/OpenRefine/issues/3514) ). Now I come back to the 3.0 version.

I hope it helps, but checking the workspace I think the problem might be the procedure when migrating from versions, I didn't make the copy of the workspace, I simply installed the new version and it gets connected with my older projects automatically (apparently).

[workspace.zip](https://github.com/OpenRefine/OpenRefine/files/6025160/workspace.zip)
